content/en/serverless/guide/serverless_tracing_and_bundlers.md

```diff
@@ -126,6 +126,113 @@ Datadog's tracing libraries (`dd-trace`) are known to be not compatible with bun
     module.exports = [nodeExternalsPlugin()]
     ```
 
+## AWS CDK & esbuild
+
+The `NodeJsFunction` construct in the AWS CDK uses esbuild. The default configuration is not compatible with Datadog's tracing libraries. The CDK allows you to override the default configuration and provide a custom esbuild file to support bundling and the Datadog tracing libraries:
+
+1. Follow the installation instructions for Node.js and ensure the Datadog Lambda layer for Node.js is added to your Lambda function.
+2. Remove `datadog-lambda-js` and `dd-trace` from your `package.json` and the build process, since they are already available in the Lambda runtime provided by the Datadog Lambda Layer.
+3. Create an `esbuild` file for each of your Lambda functions. A seperate `esbuild` file is required per Lambda function so that each entry point can be specified seperately. Notice the `entryPoint` and `outfile` properties. For example, if you had a second Lambda function in your project named `producer`, then the entry point would be `./functions/producer.ts` and the outfile would be `/out/producer/index.js`
+
+    **buildConsumer.js**
+    ```javascript
+    const ddPlugin = require('dd-trace/esbuild')
+    const esbuild = require('esbuild')
+    
+    esbuild.build({
+      entryPoints: ['./functions/consumer.ts'],
+      outfile: 'out/consumer/index.js',
+      plugins: [ddPlugin],
+      // Other esbuild configuration
+      external: [
+        // esbuild cannot bundle native modules
+        '@datadog/native-metrics',
+    
+        // required if you use profiling
+        '@datadog/pprof',
+    
+        // required if you use Datadog security features
+        '@datadog/native-appsec',
+        '@datadog/native-iast-taint-tracking',
+        '@datadog/native-iast-rewriter',
+    
+        // required if you encounter graphql errors during the build step
+        'graphql/language/visitor',
+        'graphql/language/printer',
+        'graphql/utilities',
+        '@aws-sdk/client-sqs'
```

Comment:
Let's add another entry here that is something like
```
// if you are using the package, instead of the layer
'datadog-lambda-js'
```

This suggestion also applies to the other ones in this file